publish-cli/src/main/java/com/ericsson/eiffel/remrem/publish/cli/VersionService.java

```diff
@@ -0,0 +1,131 @@
+/*
+    Copyright 2017 Ericsson AB.
+    For a full list of individual contributors, please see the commit history.
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+package com.ericsson.eiffel.remrem.publish.cli;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.jar.Attributes;
+import java.util.jar.JarFile;
+import java.util.jar.Manifest;
+
+import com.google.gson.JsonParser;
+
+
+/**
+ * This class will search in all registered jars and their manifest file for
+ * attribute Remrem-Version-Key. It will return a list with all versions found.
+ * 
+ * @author evasiba
```

Comment:
Please check and remove if we are not keeping the author's info

Reply:
Removed author's info